otelcollector/configmapparser/default-prom-configs/acstorCapacityProvisionerDefaultFile.yml

```diff
@@ -0,0 +1,47 @@
+scrape_configs:
+- job_name: acstor-capacity-provisioner
+  honor_labels: true
+  scrape_interval: $$SCRAPE_INTERVAL$$
+  scheme: http
+  kubernetes_sd_configs:
+    - role: pod

```

Comment:
isn't capacity provisioner running in a specific namespace? If so, can we just filter this discovery to that namespace only ? Can you also please check for this pattern in other discoveries in this PR ?

Reply:
never mind i see it in the later part...